project/src/main/java/com/google/sps/servlets/RegistrationServlet.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.import java.io.IOException;
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Optional;
+import com.google.sps.data.DataAccessor;
+import com.google.sps.data.UserVerifier;
+import com.google.common.annotations.VisibleForTesting;
+
+ /**
+ * A servlet that registering a user, according to a given token.
+*/
+@WebServlet("/register")
+public final class RegistrationServlet extends HttpServlet {
+
+  private static final long serialVersionUID = 1L;
+  private UserVerifier userVerifier;
+  private DataAccessor dataAccessor;
+
+  @Override
+  public void init() {
+    this.userVerifier = UserVerifier.create("CLIENT_ID");
```

Comment:
Why is this constant always passed? Is this "A Google API client ID"?

Reply:
It's like an environment variable (that holds our private credentials), but so far we haven't managed to really use it that way, so it is changed manually when we run the app.

Reply:
Ohhh Ok I didn't understand that.